README.md

```diff
@@ -1,3 +1,108 @@
-# Spark Fat Jar Template
+# Setting up a Scheduled Spark Scala job on AWS
 
-Read me will contain article in future. 
\ No newline at end of file
+Have you written a Scala Spark job that processes a massive amount of data on a stupid amount of RAM and you want to figure out a way to run it daily/weekly/monthly on a schedule on [AWS](https://aws.amazon.com)?
```

Comment:
I wasn't sure if stupid meant it was using too much or it just uses so much you wouldn't believe how much it uses. You could indicate which it is by tying the second part of the sentence back to it.